src/Microsoft.ML.Core/Data/RoleMappedSchema.cs

```diff
@@ -363,45 +380,82 @@ public ColumnInfo GetUniqueColumn(ColumnRole role)
         }
 
         /// <summary>
-        /// Creates a RoleMappedSchema from the given schema with no column role assignments.
+        /// Constructor from the given schema and role/column-name pairs.
```

Comment:
>role/column-name [](start = 50, length = 16)

I can read it as role | column-name pairs or
(role or column) | name pair.
Which probably show my lack of English, but still can it be rephrased?

Reply:
Hmmm OK. I guess I could omit names since columns are always identified by name, that might make clear I'm not talking about the strings used to construct the roles in the first place.